docs/MigrationGuide.md

```diff
@@ -3,6 +3,14 @@
 
 To see the biggest differences please consult the [changelog](Changelog.md).
 
+## In version 1.10.0
+
+### Code API changes
+* [#4278](https://github.com/TouK/nussknacker/pull/4278) Now expression compiler and code suggestions mechanism are reusing the same
+  types extracted based on model. Before the change types in compiler were lazily extracted. Because of this change, some expressions
+  can stop to compile. You may need to add `WithExplicitTypesToExtract` to some of yours `SourceFactory`. See extending classes
```

Comment:
"some of yours `SourceFactory`" -> "some of your `SourceFactory` implementations"